.github/workflows/e2e_openssl_baremetal.yml

```diff
@@ -0,0 +1,55 @@
+name: e2e test openssl baremetal
+
+on:
+  workflow_dispatch:
+    inputs:
+      skip-undeploy:
+        description: "Skip undeploy"
+        required: false
+        type: boolean
+        default: false
+  pull_request:
+    paths-ignore:
+      - dev-docs/**
+      - docs/**
+      - rfc/**
+
+env:
+  container_registry: ghcr.io/edgelesssys
+  DO_NOT_TRACK: 1
+
+jobs:
+  test:
+    runs-on:
+      labels: snp
```

Comment:
If I understand [this section](https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#example-using-a-multi-dimension-matrix) in the docs correctly, this might work.

Reply:
I think this would enable us to create tests that run on all platforms unconditionally, but still not one test that runs on one, selectable platform

Reply:
You can also run steps [conditionally](https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsif).

Reply:
Yeah, but this won't be a step. We could execute everything but the actual step on GH-hosted runners and then transfer files over, but I fear that this is going to have a higher total cost in the end.